### PR TITLE
Fix Clippy 1.77 lints

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: stratis-storage/project#697

### CODE REVIEW

**Comment by bgurney-rh** [comment]
No clippy errors in loopdev-3, libcryptsetup-rs, or libblkid-rs.  (The loopdev-3 Makefile did not have "fmt" or "fmt-ci" targets, so I added them.)

**Review by mulkieran — APPROVED** [review]

**Comment by mulkieran** [comment]
All changes good so far.

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: src/bin/stratis-min/stratisd-min.rs [hunk 1]
```diff
@@ -48,6 +48,7 @@ fn trylock_pid_file() -> StratisResult<File> {
         .read(true)
         .write(true)
         .create(true)
+        .truncate(true)
```
[the remark this answers is not in the document]
Comment (bgurney-rh) [reply src/bin/stratis-min/stratisd-min.rs]: Resolved.

File: src/bin/stratisd.rs [hunk 2]
```diff
@@ -56,6 +56,7 @@ fn trylock_pid_file() -> StratisResult<File> {
         .read(true)
         .write(true)
         .create(true)
+        .truncate(true)
```
[the remark this answers is not in the document]
Comment (bgurney-rh) [reply src/bin/stratisd.rs]: Resolved.

File: src/bin/stratis-min/stratisd-min.rs [hunk 3]
```diff
@@ -79,6 +80,7 @@ fn trylock_pid_file() -> StratisResult<File> {
...
```
[the remark this answers is not in the document]
Comment (bgurney-rh) [reply src/bin/stratis-min/stratisd-min.rs]: Resolved.

File: src/bin/stratisd.rs [hunk 4]
```diff
@@ -85,6 +86,7 @@ fn trylock_pid_file() -> StratisResult<File> {
...
```
[the remark this answers is not in the document]
Comment (bgurney-rh) [reply src/bin/stratisd.rs]: Resolved.

**Comment by bgurney-rh** [comment]
Squashed the commits for `suspicious_open_options` for stratisd and stratisd-min.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**